src/mpyl/build.py

```diff
@@ -125,22 +125,33 @@ def run_mpyl(
 
 
 def find_build_set(
-    repo: Repository, changes_in_branch: list[Revision], build_all: bool
+    repo: Repository,
+    changes_in_branch: list[Revision],
+    build_all: bool,
+    selected_stage: Optional[str] = None,
```

Comment:
Nit: maybe the word `stage` is enough?

Reply:
I used `selected_stage` on purpose to make the code on line 142 more clear and explicit on what's happening